docs/swift-sdk/integrate-with-okta.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Integrate With Okta
+sidebar_position: 2
+---
+
+This guide describes how to configure Beyond Identity using Okta as the OIDC authentication flow. 
+
+### Prerequisites
+
+ - [Integrate With Okta](../sso-integrations/integrate-with-okta)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](#authorize-with-okta), we must [Authenticate With Okta](integrate-with-okta#authorize-with-okta)
+
+### Authorize With Okta
+
+ - Step 1: Okta URL
+
+To begin the authentication flow, start an `ASWebAuthenticationSession`, and load the OAuth2 authorization request URL provided by Okta.
+
+```swift
+let session = ASWebAuthenticationSession(
+    url: viewModel.oktaURL,
+    callbackURLScheme: viewModel.callbackScheme
+    completionHandler: { (url, error) in }
+)
+session.presentationContextProvider = self
+session.start()
+```
+
+ - Step 2: Invoke URL
+
+During the session completionHandler, a URL with the invoke URL scheme should be returned from Okta. When the webpage loads a URL, call `Embedded.shared.authenticate`.
+
+```swift
+let session = ASWebAuthenticationSession(
+    url: viewModel.oktaURL,
+    callbackURLScheme: viewModel.callbackScheme
+){ (url, error) in
+    Embedded.shared.authenticate(
```

Comment:
I would also use `Embedded.shared.isAuthenticateUrl`. I know we expect the next URL to be a BI URL, but we want to:
- keep making developers aware of this functionality everywhere in our documentation
- be a bit safer in the off chance that a developer's flow is more complex that it requires this routing up front.

Reply:
`Embedded.shared.authenticate` calls `isAuthenticateUrl` under the hood so if they pass a non BI URL they will get an invalidURL error. 
But we could use this as an opportunity to document `isAuthenticateUrl` and I guess it doesn't hurt to call that function twice.